docs/src/pages/system/properties/properties.md

```diff
@@ -1,60 +1,91 @@
-| [spacing](/system/spacing/)       | `spacing`        | `pb,paddingBottom` | `padding-bottom`                                                                             | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [spacing](/system/spacing/)       | `spacing`        | `pl,paddingLeft`   | `padding-left`                                                                               | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [spacing](/system/spacing/)       | `spacing`        | `pr,paddingRight`  | `padding-right`                                                                              | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [spacing](/system/spacing/)       | `spacing`        | `pt,paddingTop`    | `padding-top`                                                                                | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [spacing](/system/spacing/)       | `spacing`        | `px,paddingX`      | `padding-left`, `padding-right`                                                              | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [spacing](/system/spacing/)       | `spacing`        | `py,paddingY`      | `padding-top`, `padding-bottom`                                                              | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [typography](/system/typography/) | `typography`     | `typography`       | `font-family`, `font-weight`, `font-size`, `line-height`, `letter-spacing`, `text-transform` | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontFamily`     | `fontFamily`       | `font-family`                                                                                | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontSize`       | `fontSize`         | `font-size`                                                                                  | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontWeight`     | `fontWeight`       | `font-weight`                                                                                | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `textAlign`      | `textAlign`        | `text-align`                                                                                 | none                                                                   |
+<p class="description">This page lists all custom system properties, how are they linked with the theme, and which CSS properties are they resulting in.</p>
```

Comment:
```suggestion
<p class="description">This page lists all custom system properties, how are they linked with the theme, and which CSS properties they compute.</p>
```

Need to read the sentence over and over. Didn't "feel" like correct english to me.